docs/changelog.md

```diff
@@ -5,6 +5,7 @@ dev
 - [bugfix] Replaced implicit dependency on setuptools with an explicit dependency on packaging (#339).
 - [refactor] Moved all commands to separate files within the commands module (#255).
 - [bugfix] Continue reinstalling packages after failure
+- [bugfix] Fixed incompatibility with Debian/Ubuntu changes made to their version of pip (#386).
```

Comment:
Considering the bug is within Debian packaging I consider this more an enhancement than bug fix; something along the line (Support Debian 20s pip patching).

Reply:
I've checked the "Allow edits from maintainers" box.

I'm a bit confused about the phrase "Support Debian 20s pip".  What's "20s"?  pip version?

I haven't mentioned specific Debian/Ubuntu/pip versions because I haven't done the research to figure out where the incompatibility got introduced.  I know things worked in Ubuntu 19.10 and broke in 20.04.  I don't run Debian myself and have trouble remembering its version numbers and codenames.

Reply:
Would 
```suggestion
- [enhancement] Insulate from problems caused by Debian/Ubuntu changes made to their version of pip (#386).
```
work for you?

Reply:
Debian decided lately to package pip by trying to devendor six from within pip... the problem is the way they did this unvendoring does not work with venvs... Not to mention the pip within the venv is supposed to be the latest pip (or the bundled version), but either way not a reference to the operating system pip, but a full copy on its own that install/uninstallable. 

Anyways I've purposefully did not do any changes here as IMHO this should be fixed on Debian side and not our end. Just throwing my ideas out there, however not planning to accept this PR myself. Maybe other maintainers are more agreeable to it, in which case I'll not stop it 👍

Reply:
The optimal solution is for Debian to fix this issue. But from a user perspective, it looks like pipx is broken, and there is no way for us to control Debian. Is there a timeline for the fix on the Debian side?

Reply:
Do you have a link to the bug in the Debian bug tracker?  (This assumes anyone reported it to Debian already.)

Once it's fixed in Debian, it should be possible to backport the fix to Ubuntu 20.04 LTS by creating a SRU.  It's just tedious work.

In the meantime I suspect users will keep filing duplicate bug reports or looking for alternative projects...

Reply:
Doesn't look like anyone's reported it to Debian, so I went ahead and did just now: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=958764